src/main/java/menion/android/whereyougo/MainApplication.java

```diff
@@ -52,7 +52,6 @@
     private static final String TAG = "MainApplication";
 
     private static Timer mTimer;
-    private static OnAppVisibilityChange onAppVisibilityChange;
```

Comment:
The changes in this file are independent from the other files.
Please extract this file and create an owh PR.
(e.g. "remove unimplemented code")